packages/cspell-json-reporter/out.json

```diff
@@ -0,0 +1,42 @@
+{
+    "issues": [
```

Comment:
Is this a sample file? Or a result of running a test?

Reply:
this a result of running example, I've forgotten to put in in gitignore

Reply:
Ahh, it is best to write it into `temp`, some of the other packages do the same thing.